### PR TITLE
Fix LookupCoordinatorManager and Test for alerts

### PR DESCRIPTION
**Pull request by drcrallen**
* Fixes test that missed https://github.com/druid-io/druid/pull/2741/files#r57495018
* Also fixes spurious alerting on Lookup delete success

### CODE REVIEW

**Comment by fjy** [comment]
@drcrallen this fails UT

File: server/src/test/java/io/druid/server/lookup/cache/LookupCoordinatorManagerTest.java [hunk 1]
```diff
@@ -431,6 +459,9 @@ void updateAllOnHost(final URL url, Map<String, Map<String, Object>> updatedLook
     };
     // Should log and pass io exception
     manager.updateAllOnTier(LOOKUP_TIER, SINGLE_LOOKUP_MAP);
+    SERVICE_EMITTER.flush();
+    Assert.assertEquals(1, EVENT_EMITS.get());
+    EVENT_EMITS.set(0);
```
Comment (gianm) [inline server/src/test/java/io/druid/server/lookup/cache/LookupCoordinatorManagerTest.java]: this happens in `setUp` so should not be necessary here

**Comment by gianm** [comment]
:+1:

File: server/src/test/java/io/druid/server/lookup/cache/LookupCoordinatorManagerTest.java (hunk 1 above)
Comment (drcrallen) [inline server/src/test/java/io/druid/server/lookup/cache/LookupCoordinatorManagerTest.java]: This is for tearDown when it checks that there are 0 events.

Comment (gianm) [inline server/src/test/java/io/druid/server/lookup/cache/LookupCoordinatorManagerTest.java]: ah ok

**Pull request merged**